apps/search/style/search.css

```diff
@@ -291,3 +290,12 @@ gaia-grid {
 [data-loading=true] .loading {
   display: block;
 }
+
+html[dir="rtl"] #suggestions-provider {
+  padding: 0 1rem 0 2rem;
```

Comment:
I see. Thanks for the investigation. It might be worth a comment pointing to that bug.